### DISCUSSION TITLE
Issue in StormBreaker with Ngrok

### DISCUSSION THREAD

**Discussion by AmitBhaiyaG** [coder/code-server#6228] [Q&A]
After running- 'python3 st.py', it says to run ngrok at Port 2525. And I run my ngrok at Port 2525 but the link doesn't work.
I have saved the ngrok authentication code and I have also run apache2.
So, PLEASE help me in details.
And also how can I change my default port in termux with step by step guide.
![IMG_20230522_104004](https://github.com/coder/code-server/assets/134181941/fbdf25dd-7c25-4f6c-aaba-48543b8b1a2d)
![Screenshot_2023-05-22-10-38-44-86_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/coder/code-server/assets/134181941/bd723774-5af7-4238-ae58-f1e5beb33665)
![Screenshot_2023-05-22-10-38-06-95_84d3000e3f4017145260f7618db1d683](https://github.com/coder/code-server/assets/134181941/11c9aebc-de8b-42e2-8cd5-f737587f4413)

**Comment by code-asher** [comment]
You might have better luck asking this in the Storm Breaker repository instead; this seems to be unrelated to code-server?
